android/expoview/src/main/java/host/exp/exponent/kernel/services/linking/LinkingKernelService.java

```diff
@@ -14,9 +14,11 @@ public LinkingKernelService() {
   }
 
   public void openURI(Uri uri) {
-    KernelProvider.getInstance().openExperience(
-        new KernelConstants.ExperienceOptions(uri.toString(), uri.toString(), null)
-    );
+    String manifestUrl = (Constants.INITIAL_URL != null && !Constants.INITIAL_URL.isEmpty()
```

Comment:
```suggestion
    String manifestUrl = (Constants.isStandaloneApp()
```
Both options would achieve the same thing but I think this is more meaningfully correct, & would not break in the future if for example we start including an `INITIAL_URL` in the client 🤷‍♂